src/windows-emulator/registry/registry_manager.cpp

```diff
@@ -105,7 +105,22 @@ std::optional<registry_key> registry_manager::get_key(const utils::path_key& key
         return {std::move(reg_key)};
     }
 
-    const auto* entry = iterator->second->get_sub_key(reg_key.path.get());
+    auto path = reg_key.path.get();
+    const auto* entry = iterator->second->get_sub_key(path);
+
+    if (!entry)
+    {
+        constexpr std::wstring_view wowPrefix = L"wow6432node\\";
+
+        const auto pathStr = path.wstring();
```

Comment:
isn't that only done for 32 bit applications?

Reply:
I assume you’ve already found the answer, but x64 applications can request that key by using the KEY_WOW64_32KEY flag.